docs/document-level-security/SALESFORCE.md

```diff
@@ -0,0 +1,57 @@
+### Setting up the Salesforce connector
+
+See the [Developer guide](../../docs/DEVELOPING.md) for setting up connectors.
+
+## Document level security
+
+Document level security (DLS) enables you to restrict access to documents based on a user'­s permissions. This feature is available by default for the Salesforce connector.
+Salesforce connector DLS supports for both standard & custom objects.
+
+Refer to [document level security](https://www.elastic.co/guide/en/enterprise-search/master/dls.html) for more information.
```

Comment:
Should this link to /current/ ?

Reply:
I took the reference from already merged connector DLS documentation. Do we need to update this link?

Reply:
We're on main here so master in the main docs makes sense